### ISSUE TITLE
exercise "Split data into training, validation, and test set" not very interesting

### ISSUE THREAD

**Issue by wikfeldt** [carpentries-incubator/deep-learning-intro#198]
we skipped over it in the ENCCS workshop because it's a basic arithmetic exercise that doesn't add any understanding about neural networks

**Comment by dafnevk** [comment]
I understand that, given also limited amount of time, this may not be the most interesting exercise. I think we mostly included it to make sure learners interact with the data, and are aware of the workflow step of creating train, test and validation sets.  Currently, episode 3 has very few exercises (see issue #183 and issue #190), maybe once we add some more interesting exercises we can remove this one.
